src/components/ChangeLocation/index.js

```diff
@@ -28,6 +29,11 @@ const ChangeLocation = ({
     }
   }, [show]);
 
+  const buttonLabel = (buttonType === 'Save'
+    ? 'Save'
+    : 'Apply'
+  );
+
```

Comment:
we can pass a buttonLabel prop instead of buttonType and pass the buttonLabel prop directly to the Button component